app/services/user_updater.rb

```diff
@@ -180,8 +180,8 @@ def update(attributes = {})
       end
     end
 
-    if attributes.key?(:skip_new_user_tips)
-      user.user_option.seen_popups = user.user_option.skip_new_user_tips ? [-1] : nil
+    if attributes.key?(:skip_new_user_tips) && user.user_option.skip_new_user_tips
```

Comment:
Since the behaviour changed, we should also change the specs to ensure there's only one way to skip the new user tips